BYT/View Controllers/FoaasViewController.swift

```diff
@@ -120,6 +124,20 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
         }
         self.foaasView.mainTextLabel.text = validFoaas.message
         self.foaasView.subtitleTextLabel.text = validFoaas.subtitle
+
+        DispatchQueue.main.async {
+            let valueForConstraintConstant = validFoaas.subtitle.characters.count
+            if self.foaasView.subtitleLabelConstraint.constant - CGFloat(Double(valueForConstraintConstant) * 1.5) < 16 {
```

Comment:
Sorry for the confusing constant name. I just updated it. Please see newest commit.